src/tests/api/Filter.test.js

```diff
@@ -0,0 +1,136 @@
+import { applyFilter } from 'src/api';
+import { filters } from './data-mocks/filters';
+
+const makePossibleFeature = (value) => [
```

Comment:
A bit strange name... 😄  ? how about makeFeatureWithValueInColum? 

Also I find a bit strange to use 'column1' as the default column... is there a reason for this?
If there is no special requirement, we could add a new parameter, and modify a bit the function like:

```
const makeFeatureWithValueInColum = (value, column='column1')
```

What do you think?